### ISSUE TITLE
Git update-git-for-windows -y causes Installer Popup

### ISSUE THREAD

**Issue by ld0614** [git-for-windows/git#2548]
- [x]  [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

2.23.0.windows.1 (64bit)
(since updated to 2.25.1)
```
$ git --version --bild-options

git version 2.25.1.windows.1
cpu: x86_64
built from commit: cceb69da75b4906bb04cfb3a2f7f37b12b985ec3
sizeof-long: 4
sizeof-size_t: 8

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.18362.720]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VisualStudioCode
Custom Editor Path: 
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: WinSSL
CRLF Option: CRLFAlways
Bash Terminal Option: ConHost
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

No

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

CMD or PowerShell

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
git update-git-for-windows -y
```
 - What did you expect to occur after running these commands?

git should automatically identify and update, download it and silently install it

 - What actually happened instead?

git successfully identifies and downloads the update however when it triggers the update it causes a license screen to display which requires user interaction.  Therefore the update can't be fully automated.  

I believe that this is due to the installer only calling the /SILENT command on the new installer. 
 When manually testing with the /VERYSILENT command the installer works as expected.  I believe that the triggering code is [here](https://github.com/git-for-windows/build-extra/blob/a97ce2ddf420ab4456111a03af2d51f3347eadae/git-extra/git-update-git-for-windows#L293)

I'm happy to create a PR just to update this parameter but didn't know if this was desired behavior / this should be behind a separate update parameter?

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

N/A

**Comment by dscho** [comment]
> I believe that this is due to the installer only calling the /SILENT command on the new installer.
> When manually testing with the /VERYSILENT command the installer works as expected. I believe that the triggering code is [here](https://github.com/git-for-windows/build-extra/blob/a97ce2ddf420ab4456111a03af2d51f3347eadae/git-extra/git-update-git-for-windows#L293)

Good idea!

> I'm happy to create a PR just to update this parameter but didn't know if this was desired behavior / this should be behind a separate update parameter?

Yes, please create a PR, and yes, what you described is the desired behavior.

**Comment by dscho** [comment]
Fixed via git-for-windows/build-extra#279
